src/io.c

```diff
@@ -86,7 +92,9 @@ unsigned char io_event(unsigned char channel MARK_UNUSED) {
             break;
 
         case SEPROXYHAL_TAG_DISPLAY_PROCESSED_EVENT:
+#ifdef HAVE_BAGL
             UX_DISPLAYED_EVENT({});
+#endif
```

Comment:
UX_DEFAULT_EVENT for NBGL ?
Copy this function from boilerplate

Reply:
Done.